src/signals/incident/containers/KtoContainer/components/KtoForm/index.js

```diff
@@ -25,41 +24,39 @@ class KtoForm extends React.Component { // eslint-disable-line react/prefer-stat
     this.ktoForm = ktoDefinition;
   }
 
-  UNSAFE_componentWillReceiveProps(props) {
+  componentDidUpdate(props) {
     if (!isEqual(props.ktoContainer.answers, this.props.ktoContainer.answers) && this.ktoForm && this.ktoForm.controls) {
-      if (props.ktoContainer.form.is_satisfied && this.ktoForm.controls.tevreden && this.ktoForm.controls.tevreden.meta) {
+      if (this.props.ktoContainer.form.is_satisfied && this.ktoForm.controls.tevreden && this.ktoForm.controls.tevreden.meta) {
```

Comment:
Why reference `this.props` where before it was `props`?

Reply:
because props is actually prevProps now in react 16